### PR TITLE
Updated WandbLogger to make uploading model checkpoints optional

### PR DESCRIPTION
**Pull request by manangoel99**
This PR adds a minor patch to the WandbLogger integration to make saving model checkpoints to the W&B dashboard optional. To save checkpoints to the dashboard, the users just need to update the yaml file like so

```
wandb:
    project: my_project
    entity: my_team
    log_checkpoint: true
```

Logging checkpoints is turned off by default.

### CODE REVIEW

**Comment by manangoel99** [comment]
I was wondering if I could get some feedback on this PR whenever possible

**Comment by manangoel99** [comment]
Hi @LDOUBLEV @WenmuZhou I was wondering if it would be possible to get a review on this PR

File: ppocr/utils/loggers/wandb_logger.py [hunk 1]
```diff
@@ -40,12 +44,14 @@ def __init__(self,
...
             self.run.config.update(self.config)
+        
+        self.log_checkpoint = log_checkpoint
 
     @property
     def run(self):
         if self._run is None:
             if self.wandb.run is not None:
-                logger.info(
+                logging.info(
```
Comment (WenmuZhou) [inline ppocr/utils/loggers/wandb_logger.py:54]: It is recommended to use the inner logger to print, which can be initialized with the following code
```python
from ppocr.utils.logging import get_logger
logger = get_logger()
```

**Comment by TuanBC** [comment]
@manangoel99 This pull request is actually very useful for many use cases that I'm using. Could you review this pull request again based on the above comment?

**Comment by manangoel99** [comment]
@WenmuZhou I've made the requested change

**Pull request closed without merging**